### PR TITLE
Add editor to representer feedback

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Screenshot 2023-03-31 at 14 51 57

### CODE REVIEW

File: app/models/submission.rb [hunk 1]
```diff
@@ -159,7 +160,15 @@ def representer_feedback
         reputation: author.reputation,
         avatar_url: author.avatar_url,
         profile_url: author.profile ? Exercism::Routes.profile_url(author) : nil
-      }
+      },
+      editor: if editor.present?
```
Comment (ErikSchierboom) [inline app/models/submission.rb:164]: This is rubocop's doing (the if/end)

**Review by iHiD — APPROVED** [review]
Nice.

File: app/javascript/components/student/iterations-list/RepresenterFeedback.tsx [hunk 2]
```diff
@@ -21,3 +26,16 @@ export const RepresenterFeedback = ({ html, author }: Props): JSX.Element => {
     </div>
   )
 }
+
+function EditedBy({
+  author,
+  editor,
+}: Pick<Props, 'author' | 'editor'>): JSX.Element | null {
+  if (editor && editor.name !== author.name) {
```
Comment (iHiD) [inline app/javascript/components/student/iterations-list/RepresenterFeedback.tsx]: Maybe early return is nicer here?
`if (!editor || editor.name == author.name) { return null }`

File: app/javascript/components/student/iterations-list/RepresenterFeedback.tsx [hunk 3]
```diff
@@ -31,11 +31,13 @@ function EditedBy({
   author,
   editor,
 }: Pick<Props, 'author' | 'editor'>): JSX.Element | null {
-  if (editor && editor.name !== author.name) {
+  if (!editor || editor.name === author.name) {
+    return null
+  } else {
```
Comment (iHiD) [inline app/javascript/components/student/iterations-list/RepresenterFeedback.tsx]: You don't need this else now (that's the beauty of the early return is that everything else stops needing to be indented).

**Pull request merged**